js/client/modules/@arangodb/testsuites/recovery.js

```diff
@@ -100,12 +100,8 @@ function runArangodRecovery (instanceInfo, options, script, setup, count) {
   ]);
 
   let binary = pu.ARANGOD_BIN;
-  if (setup) {
-    binary = pu.TOP_DIR + '/scripts/disable-cores.sh';
-    argv.unshift(pu.ARANGOD_BIN);
-  }
 
-  instanceInfo.pid = pu.executeAndWait(binary, argv, options, 'recovery', instanceInfo.rootDir, setup, !setup && options.coreCheck);
+  instanceInfo.pid = pu.executeAndWait(binary, argv, options, 'recovery', instanceInfo.rootDir, setup, setup, !setup && options.coreCheck);
```

Comment:
This does not look right. pu.executeAndWait is called with 8 arguments but takes only 7. I think there was a bug before already, which should now be fixed.